src/Collection/AssetsRecord.php

```diff
@@ -0,0 +1,18 @@
+<?php
+declare(strict_types=1);
+
+namespace PayCrypto\Collection;
+
+use ScriptFUSION\Porter\Collection\CountableProviderRecords;
+use ScriptFUSION\Porter\Provider\Resource\ProviderResource;
+
+class AssetsRecord extends CountableProviderRecords
```

Comment:
This class and the `ExchangesRecord` class doesn't do anything and should be removed. Just invoke `new CountableProviderRecords` directly.

Reply:
Can you show some sample about the invoke ```CountableProviderRecords``` class directly?

Reply:
I see that. Using the ```CountableProviderRecords``` directly because it has no any additional argument to be assigned.